manager/controlapi/service.go

```diff
@@ -378,6 +379,11 @@ func (s *Server) UpdateService(ctx context.Context, request *api.UpdateServiceRe
 		if reflect.TypeOf(service.Spec.Mode) != reflect.TypeOf(request.Spec.Mode) {
 			return errModeChangeNotAllowed
 		}
+
+		if service.Spec.Annotations.Name != request.Spec.Annotations.Name {
+			return errRenameNotSupported
```

Comment:
It just occurred to me that it might be helpful to show the old and new name of the service, in case a user (especially an API user) encounters this unexpectedly. WDYT? I don't think it's very important though, so feel free to ignore this suggestion.